Yosemite/Yosemite/Stores/DataStore.swift

```diff
@@ -0,0 +1,99 @@
+import Foundation
+import Networking
+import Storage
+
+// MARK: - DataStore
+//
+public final class DataStore: Store {
```

Comment:
Same as `DataAction` above - name without context is too generic and confusing, consider clarifying in a comment, maybe with docs link.